cmd/influx/main.go

```diff
@@ -73,32 +75,11 @@ func out(w io.Writer) genericCLIOptfn {
 	}
 }
 
-func fetchSubCommand(parent *cobra.Command, args []string) *cobra.Command {
-	var err error
-	var cmd *cobra.Command
-
-	// Workaround FAIL with "go test -v" or "cobra.test -test.v", see #155
-	if args == nil && filepath.Base(os.Args[0]) != "cobra.test" {
-		args = os.Args[1:]
-	}
-
-	if parent.TraverseChildren {
-		cmd, _, err = parent.Traverse(args)
-	} else {
-		cmd, _, err = parent.Find(args)
-	}
-	// return nil if any errs
-	if err != nil {
-		return nil
-	}
-	return cmd
-}
-
-func seeHelp(c *cobra.Command, args []string) {
-	if c = fetchSubCommand(c, args); c == nil {
-		return //return here, since cobra already handles the error
-	}
-	c.Printf("See '%s -h' for help\n", c.CommandPath())
+var flags struct {
```

Comment:
i don't really like such a global thing.

Reply:
totally agree. I didn't remedy all the issues in this pkg on this go round. However, I agree, the global constructs in this are a major issue and should be fixed. I didn't do it here b/c it was already bigger body of work than I anticipated 😢